save-preprocessor/src/main/kotlin/com/saveourtool/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -126,29 +117,114 @@ class DownloadProjectController(
     @PostMapping(value = ["/uploadBin"])
     fun uploadBin(
         @RequestBody executionRequestForStandardSuites: ExecutionRequestForStandardSuites,
-    ) = Mono.just(ResponseEntity(executionResponseBody(executionRequestForStandardSuites.executionId), HttpStatus.ACCEPTED))
+    ) = executionResponseAsMono(executionRequestForStandardSuites)
         .doOnSuccess {
-            val version = requireNotNull(executionRequestForStandardSuites.version) {
-                "Version is not provided for execution.id = ${executionRequestForStandardSuites.executionId}"
-            }
-            val execCmd = executionRequestForStandardSuites.execCmd
-            val batchSizeForAnalyzer = executionRequestForStandardSuites.batchSizeForAnalyzer
-            getStandardTestSuiteIds(executionRequestForStandardSuites.testSuites)
-                .flatMap { testSuiteIds ->
-                    updateExecution(
-                        executionRequestForStandardSuites.project,
-                        "N/A",
-                        version,
-                        testSuiteIds,
-                        execCmd,
-                        batchSizeForAnalyzer,
+            Flux.fromIterable(standardTestSuitesRepo.testRootPaths)
```

Comment:
We don't update standard test suites on each run in current logic, need to spik fetching here

Reply:
added search of latest version